WordPress/JetpackScreenshotGeneration/JetpackScreenshotGeneration.swift

```diff
@@ -31,7 +32,7 @@ class JetpackScreenshotGeneration: XCTestCase {
         super.tearDown()
     }
 
-    func testGenerateScreenshots() {
+    func testGenerateScreenshots() throws {
```

Comment:
Because the `ScreenObject` `init` `throws`, we now have a sprinkle of `try` calls around the test suite.

That's good. XCTest knows how to fail tests and report errors if a `try` call `throws`.